resources/views/partials/language-switcher.blade.php

```diff
@@ -1,7 +1,5 @@
-<language-switcher language="{{ $language }}" class="pb-4 md:pb-0">
-    @foreach ($localeSlugs as $locale)
-        <a href="{{ switch_locale_link($locale) }}"
-           class="block px-4 py-2 text-blue-700 hover:bg-blue-700 hover:text-white"
-           style="text-decoration: none">{{ Localization::languageForLocale($locale) }}</a>
-    @endforeach
+<language-switcher
+    language="{{ $language }}"
+    :other-languages="{{ json_encode($locales) }}"
```

Comment:
Sorry, one last note! Naming this prop "other languages" makes me think you've excluded this chosen language, but this is really *all* languages. Can you rename it here and in the component to just "languages" and then you can merge this PR? Thanks!